src/exit/graph.py

```diff
@@ -0,0 +1,96 @@
+from gql import gql
+from web3 import Web3
+from web3.types import ChecksumAddress, Wei
+
+from .clients import graph_client
+from .typings import LeveragePosition, OsTokenExitRequest
+
+
+def graph_get_leverage_positions() -> list[LeveragePosition]:
+    query = gql(
+        """
+        query PositionsQuery {
+          leverageStrategyPositions(
+            orderBy: borrowLtv, orderDirection: desc
```

Comment:
I think there is a max amount of entities that the subgraph returns if the `limit` is not provided. I think it's better to add pagination as in v2 oracle. Also, please check all the other graphql queries for the same potential issue.